u8timeseries/timeseries.py

```diff
@@ -20,35 +19,34 @@
 
 class TimeSeries:
     def __init__(self,
-                 series: pd.Series,
-                 confidence_lo: Optional[pd.Series] = None,
-                 confidence_hi: Optional[pd.Series] = None,
+                 series: Union[pd.Series, pd.DataFrame],
```

Comment:
Do we want to allow for both or maybe it would be easier just to assume DataFrame, and Series would be just a dataframe for univariate case (ie "1 row")?

Reply:
Yeah I thought about that too. I kept supporting it for compatibility with existing code. Thoughts on this @hrzn?

Reply:
I think we could keep only `DataFrame` here, but perhaps have a static method `from_series` that would allow to build a univariate one from a `Series`.

Reply:
done